estela-web/src/components/ProjectListPage/index.tsx

```diff
@@ -63,7 +63,7 @@ export class ProjectListPage extends Component<unknown, ProjectsPageState> {
         return (
             <Layout className="general-container">
                 <Header />
-                <Layout className="white-background">
+                <Layout className="bg-[#FBFCFD]">
```

Comment:
@raymond1242 I would prefer for this to be on a css file, because it makes it very restrictive to add it directly inside the code.